src/blocks/blocks/font-awesome-icons/style.scss

```diff
@@ -24,6 +24,7 @@
 
 		i {
 			width: 100%;
+			min-width: calc( var(--fontSize) * 1.925399 ); // 0.01 precision
```

Comment:
@HardeepAsrani, @mghenciu, if I remember correctly, the incorrect size of the icon was reported in the past. Like in the past, I did not find a complete CSS solution to solve this issue. 

The way the font scale in width is not linear with the height base on the font size. Best I can do is to multiple with a number that gives a good enough aspect for some values of the font-size.

Unfortunatly, CSS does not allow complex math functions so that I can find a more suitable formula.

One way is to solve via JS, or find an alternative way to display the icons. I will research how other plugins are doing it, hoping that they also use FA.